x-pack/solutions/observability/plugins/observability_ai_assistant_app/server/rule_connector/index.ts

```diff
@@ -64,7 +69,16 @@ const connectorParamsSchemas: Record<string, CompatibleJSONSchema> = {
 
 const ParamsSchema = schema.object({
   connector: schema.string(),
-  message: schema.string({ minLength: 1 }),
+  prompts: schema.maybe(
+    schema.arrayOf(
+      schema.object({
+        statuses: schema.arrayOf(schema.string()),
+        message: schema.string({ minLength: 1 }),
+      })
+    )
+  ),
+  status: schema.maybe(schema.string()),
+  message: schema.maybe(schema.string({ minLength: 1 })), // this is a legacy field
```

Comment:
When can this be removed? Should we add a `TODO: remove in 9.1` or something like that?